test/cpp/api/operations.cpp

```diff
@@ -55,3 +55,33 @@ TEST_F(OperationTest, Cross) {
     ASSERT_TRUE(out.allclose(exp));
   }
 }
+
+TEST_F(OperationTest, Linear_out) {
+  {
+    const auto x = torch::arange(100., 118).resize_({3, 3, 2});
```

Comment:
nitpick: can we enable the out variant in OpInfo (https://github.com/pytorch/pytorch/wiki/Writing-tests-in-PyTorch-1.8) in addition to these tests

Reply:
I can do this in a separate PR